modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -1113,13 +1109,18 @@ RETURN d.hours, d.minutes, d.seconds, d.milliseconds, d.microseconds, d.nanoseco
 6+d|Rows: 1
 |===
 
-======
+`d.minutes` is the sum of 60 minutes of the hour and the one minute from the query as both `duration.hours` and `duration.minutes` are both seconds-based components.
+Similarly, `d.seconds`, `d.milliseconds`, `d.microseconds` and `d.nanoseconds` are sum values of the relevant seconds-based components from the query.
 
+`d.hours` doesn't take the day from the query into account because `duration.days` is a days-based component.
```

Comment:
Just personal preference :) 
```suggestion
`d.hours` does not take the day from the query into account because `duration.days` is a days-based component.
```